go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -47,6 +48,13 @@ const (
 	dataDictionaryFile      = "mysql.ibd"
 )
 
+var (
+	// BuiltinBackupMysqldDeadline is how long ExecuteBackup should wait for response from mysqld.Shutdown.
```

Comment:
typically `deadline` is used for timestamps and `timeout` for durations. This looks like a timeout.